test/Grpc.IntegrationTests/IntegrationTestBase.cs

```diff
@@ -107,4 +94,31 @@ protected IReadOnlyCollection<LogEntry> GetLogs()
         Assert.True(foundCategory);
         return logs;
     }
+
+    protected struct HostLifetime : IAsyncDisposable
```

Comment:
I'm curious how you came up with the name `HostLifetime`? It confused me because of how similar it is to [IHostLifetime](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.hosting.ihostlifetime?view=dotnet-plat-ext-6.0), which I think is unrelated.

Reply:
Being test code, I wasn't too concerned with the most ideal name. Just wanted something to represent the lifetime of the host within this test.